dataprofiler/labelers/column_name_model.py

```diff
@@ -39,7 +39,8 @@ def __init__(self, parameters=None):
         parameters.setdefault("include_label", True)
         parameters.setdefault("negative_threshold_config", None)
 
-        # initialize class
+        # validate and set parameters
+        self.set_label_mapping(label_mapping)
         self._validate_parameters(parameters)
         self._parameters = parameters
```

Comment:
Instead of these, could call the super?
```python
BaseModel.__init__(self, label_mapping, parameters)
```

Reply:
maybe we can't bc regex didn't? or an oversight.

Reply:
I'll test

Reply:
I'm seeing some of the other models use the `BaseModel.__init__` but when I do that or the super unit tests are failing... I'll trouble shoot a bit more though

Reply:
Not seeing success with doing `BaseModel.__init__(self, label_mapping, parameters)` ... maybe I'm doing something wrong, though

Reply:
curious, what were the errors?